### PR TITLE
fix: stripe fees hardcoded

### LINKED ISSUE #9447: [CAL-2514] Stripe Fees are hardcoded in the component
The settings for the fee percentage are set to 0 but Stripe component still renders "0.5% fee + $0.10  commission"
image

<sub>[CAL-2514](https://linear.app/calcom/issue/CAL-2514/stripe-fees-are-hardcoded-in-the-component)</sub>

### PR DESCRIPTION
**Pull request by JaideepGuntupalli**
## What does this PR do?

Linked stripe payment and percentage env to UI.

ENV VARS CHANGED:
PAYMENT_FEE_FIXED -> NEXT_PUBLIC_PAYMENT_FEE_FIXED
PAYMENT_FEE_PERCENTAGE -> NEXT_PUBLIC_PAYMENT_FEE_PERCENTAGE

Fixes #9447 

image

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- This change might require a documentation update since the env variable name has been changed

## How should this be tested?
- Try changing env vars and adding a stripe or read its desc.

## Mandatory Tasks
- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by JaideepGuntupalli** [comment]
hey @keithwillcode, just a reminder, can you pls review the PR

**Review by alannnc — DISMISSED** [review]
Code looks good!
Will leave merge to @zomars, so he can update env vars on servers.

**Review by emrysal — DISMISSED** [review]
Big fan of this, thank you @JaideepGuntupalli

**Review by zomars — DISMISSED** [review]

File: .github/workflows/env-create-file.yml
Comment (zomars) [inline .github/workflows/env-create-file.yml:1]: This file got deleted in main

**Review by zomars — DISMISSED** [review]
Updated and approved

**Comment by PeerRich** [comment]
do we still need this, since we removed the fees? @zomars

**Comment by PeerRich** [comment]
sorry this went stale. we have removed stripe fees for now so this is not needed

**Pull request closed without merging**